<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HospitalBedFindComboBoxPopup</class>
 <widget class="QWidget" name="HospitalBedFindComboBoxPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout">
   <property name="margin">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>0</number>
   </property>
   <item row="0" column="0">
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::South</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tabHospitalBedFind">
      <attribute name="title">
       <string>Результат поиска</string>
      </attribute>
      <layout class="QVBoxLayout">
       <property name="spacing">
        <number>4</number>
       </property>
       <property name="margin">
        <number>4</number>
       </property>
       <item>
        <widget class="CHospitalBedPopupView" name="tblHospitalBedFind"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabSearch">
      <attribute name="title">
       <string>&amp;Поиск</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <property name="margin">
        <number>4</number>
       </property>
       <property name="spacing">
        <number>4</number>
       </property>
       <item row="0" column="0">
        <widget class="QLabel" name="lblOrgStructureBed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Подразделение</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" colspan="6">
        <widget class="COrgStructureComboBox" name="cmbOrgStructureBed"/>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="lblSexBed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Пол</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QComboBox" name="cmbSexBed">
         <item>
          <property name="text">
           <string>нет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>М</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Ж</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="1" column="2">
        <widget class="QLabel" name="lblAgeBed">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="text">
          <string>Возраст от</string>
         </property>
        </widget>
       </item>
       <item row="1" column="3">
        <widget class="QSpinBox" name="edtAgeForBed">
         <property name="maximum">
          <number>150</number>
         </property>
        </widget>
       </item>
       <item row="1" column="4">
        <widget class="QLabel" name="lblAgeTo">
         <property name="text">
          <string>до</string>
         </property>
        </widget>
       </item>
       <item row="1" column="5">
        <widget class="QSpinBox" name="edtAgeToBed"/>
       </item>
       <item row="1" column="6">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>33</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="lblProfileBed">
         <property name="text">
          <string>Профиль</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" colspan="6">
        <widget class="CRBComboBox" name="cmbProfileBed"/>
       </item>
       <item row="3" column="0">
        <widget class="QLabel" name="lblTypeBed">
         <property name="text">
          <string>Тип</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" colspan="6">
        <widget class="CRBComboBox" name="cmbTypeBed"/>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="lblIsPermanentBed">
         <property name="text">
          <string>Штат</string>
         </property>
        </widget>
       </item>
       <item row="4" column="1" colspan="6">
        <widget class="QComboBox" name="cmbIsPermanentBed">
         <item>
          <property name="text">
           <string>Все</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Нет</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>Да</string>
          </property>
         </item>
        </widget>
       </item>
       <item row="5" column="0" colspan="3">
        <spacer>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>231</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="5" column="3" colspan="4">
        <widget class="QDialogButtonBox" name="buttonBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="standardButtons">
          <set>QDialogButtonBox::Apply|QDialogButtonBox::Reset</set>
         </property>
         <property name="centerButtons">
          <bool>false</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>COrgStructureComboBox</class>
   <extends>QComboBox</extends>
   <header>Orgs/OrgStructComboBoxes.h</header>
  </customwidget>
  <customwidget>
   <class>CRBComboBox</class>
   <extends>QComboBox</extends>
   <header>library/crbcombobox.h</header>
  </customwidget>
  <customwidget>
   <class>CHospitalBedPopupView</class>
   <extends>QTableView</extends>
   <header>HospitalBeds/HospitalBedComboBox.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>cmbOrgStructureBed</tabstop>
  <tabstop>cmbSexBed</tabstop>
  <tabstop>edtAgeForBed</tabstop>
  <tabstop>edtAgeToBed</tabstop>
  <tabstop>cmbProfileBed</tabstop>
  <tabstop>cmbTypeBed</tabstop>
  <tabstop>cmbIsPermanentBed</tabstop>
  <tabstop>buttonBox</tabstop>
  <tabstop>tblHospitalBedFind</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
